在退出Nonebot实例时浏览器提前退出导致退出函数阻塞，Nonebot无法关闭的错误

如题 Log如下 在Windows系统下会遇到该情况

在命令退出Nonebot的时候浏览器进程就已被清除，不明白什么原因

杀自己的时候发现自己已经被杀了 愣住了）

``` 
03-06 15:31:37 [INFO] nonebot_plugin_htmlrender | Browser Starting.
03-06 15:31:38 [INFO] nonebot_plugin_htmlrender | 使用 chromium 启动
03-06 15:31:38 [INFO] nonebot_plugin_htmlrender | Browser Started.
03-06 15:31:38 [INFO] uvicorn | Application startup complete.
03-06 15:31:38 [INFO] uvicorn | Uvicorn running on http://127.0.0.1:8080 (Press CTRL+C to quit)
03-06 15:31:40 [ERROR] nonebot_plugin_htmlrender | Browser Disconnected
03-06 15:31:40 [INFO] uvicorn | Shutting down
03-06 15:31:40 [INFO] uvicorn | Waiting for application shutdown.
03-06 15:31:40 [INFO] nonebot_plugin_htmlrender | Browser Stopping.
03-06 15:31:40 [INFO] nonebot_plugin_htmlrender | shutdown_browser
03-06 15:31:40 [INFO] nonebot_plugin_htmlrender | _browser is None=False
03-06 15:31:40 [INFO] nonebot_plugin_htmlrender | _browser.is_connected()=False
```

--- a/nonebot_plugin_htmlrender/browser.py
+++ b/nonebot_plugin_htmlrender/browser.py
@@ -18,6 +18,7 @@ from nonebot.log import logger
 from playwright.async_api import Browser, Error, Page, Playwright, async_playwright
 
 from .config import Config
+import asyncio
 
 config = get_plugin_config(Config)
 
@@ -78,7 +79,7 @@ async def shutdown_browser():
             await _browser.close()
         _browser = None
     if _playwright:
-        await _playwright.stop()  # type: ignore
+        # await asyncio.wait_for(_playwright.stop(), timeout=3)
         _playwright = None
 
 

--- a/nonebot_plugin_htmlrender/browser.py
+++ b/nonebot_plugin_htmlrender/browser.py
@@ -74,7 +74,8 @@ async def shutdown_browser():
     global _browser
     global _playwright
     if _browser:
-        await _browser.close()
+        if _browser.is_connected():
+            await _browser.close()
         _browser = None
     if _playwright:
         await _playwright.stop()  # type: ignore
@@ -92,9 +93,9 @@ async def install_browser():
         os.environ["PLAYWRIGHT_DOWNLOAD_HOST"] = host
     else:
         logger.info("使用镜像源进行下载")
-        os.environ[
-            "PLAYWRIGHT_DOWNLOAD_HOST"
-        ] = "https://npmmirror.com/mirrors/playwright/"
+        os.environ["PLAYWRIGHT_DOWNLOAD_HOST"] = (
+            "https://npmmirror.com/mirrors/playwright/"
+        )
     success = False
 
     if config.htmlrender_browser == "firefox":

